client/landing/gutenboarding/constants.ts

```diff
@@ -49,5 +49,8 @@ export const fontPairings = [
 	},
 ] as const;
 
-export type FontPair = ValuesType< typeof fontPairings >;
-export type Font = FontPair[ keyof FontPair ];
+export type Font = ValuesType< ValuesType< typeof fontPairings > >;
+export interface FontPair {
+	headings: Font;
+	base: Font;
+}
```

Comment:
Let known fonts appear in either position.

Reply:
Maybe these should be moved [here](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/stores/onboard/types.ts)? Then `Design` will have the correct `fonts` types, not `array < string >`. It's 100% up to you.